src/libcublas_types.jl

```diff
@@ -61,3 +61,43 @@ const cuDoubleComplex = Complex{Float64}
 const CublasFloat = Union{Float64,Float32,Complex128,Complex64}
 const CublasReal = Union{Float64,Float32}
 const CublasComplex = Union{Complex128,Complex64}
+# FP16 (cuda_fp16.h) in cuda
+const __half = Float16;
+immutable __half2
+    x1::__half
+    x2::__half
+end
+try
```

Comment:
Why is the `try` necessary? We should also add a precompilation dependencies to `libcudart.so`.